spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/DataFlowTemplate.java

```diff
@@ -151,19 +152,19 @@ public DataFlowTemplate(URI baseURI, RestTemplate restTemplate, ObjectMapper map
 
 		if (resourceSupport != null) {
 			if (resourceSupport.getApiRevision() == null) {
-				throw new IllegalStateException("Incompatible version of Data Flow server detected.\n"
-						+ "Follow instructions in the documentation for the version of the server you are "
-						+ "using to download a compatible version of the shell.\n"
-						+ "Documentation can be accessed at https://cloud.spring.io/spring-cloud-dataflow/");
+				throw new IllegalStateException("""
```

Comment:
Oooohhh sexy multi-line strings. Thanks Java17 ;P

Reply:
Couldn't resist.

Reply:
Its full of stars!!!!!